bin/udmi-locate

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash -e
 
-UDMI_ROOT=$(realpath $(dirname $(readlink -f $0))/..)
-
-source $UDMI_ROOT/etc/shell_common.sh
-
-source $UDMI_ROOT/etc/find_udmi_profile.sh
+source $(realpath $(dirname $(readlink -f $0))/..)/etc/udmi_preamble.sh
```

Comment:
Honestly all this source'ing makes me nervous. Is there some other way we can accomplish this? 

If we just want to put configuration data into shell scripts that's one thing, and carefully pull the values out. But at the point that we are building multiple source (with sourceing them) shell scripts I think we might need a different approach.

WDYT?

Reply:
I don't know how else to do it other than with "source" -- if you have a specific suggestion on "some other way" that's more appropriate then I'm all ears... but -- if you want/need to do something like define "common functions" I'm not sure how else to do it...

Reply:
Yes, I think if you are sourcing shell scripts you need to move to a different language.